### ISSUE TITLE
models/onnx_export_my.py

### ISSUE THREAD

**Issue by ksd777** [jinfagang/yolov5#1]
Did you add your custom onnx export function? It seems there is the original one from Ultralytics only

**Comment by jinfagang** [comment]
Yeah, this repo for experiments only. internel operation will not publich at the moment.
